ci/tlk.ps1

```diff
@@ -345,6 +345,7 @@ class TlkRecipe
                 if ($this.Target.IsWindows()) {
                     $agentPackages += [TlkPackage]::new("devolutions-pedm-hook", "crates/devolutions-pedm-hook", $true)
                     $agentPackages += [TlkPackage]::new("devolutions-pedm-contextmenu", "crates/devolutions-pedm-contextmenu", $true)
+                    $agentPackages += [TlkPackage]::new("devolutions-host", "crates/devolutions-host", $false)
```

Comment:
Minor changes just to build binary; Packaging + Installer changes will be in the follow-up PR soon